pyk/src/pyk/cterm/cterm.py

```diff
@@ -375,6 +376,8 @@ def cterm_build_rule(
         init_cterm: State to put on LHS of the rule (constraints interpreted as `requires` clause).
         final_cterm: State to put on RHS of the rule (constraints interpreted as `ensures` clause).
         keep_vars: Variables to leave in the side-conditions even if not bound in the configuration.
+        priority: Priority index to use for generated rules.
+        defunc_with: KDefinition to be able to defunctionalize LHS appropriately.
```

Comment:
```suggestion
        defunc_with (optional): KDefinition to be able to defunctionalize LHS appropriately.
```

Renders like this: https://kframework.org/pyk/api/pyk.cterm.cterm.html#pyk.cterm.cterm.CTerm.anti_unify